src/common/lib/types/message.ts

```diff
@@ -9,6 +9,30 @@ import * as API from '../../../../ably';
 import { IUntypedCryptoStatic } from 'common/types/ICryptoStatic';
 import { MsgPack } from 'common/types/msgpack';
 
+const MessageActionArray: API.MessageAction[] = [
+  'message_unset',
+  'message_create',
+  'message_update',
+  'message_delete',
+  'message_annotation_create',
+  'message_annotation_delete',
+  'message_meta_occupancy',
```

Comment:
The [proposal](https://docs.google.com/document/d/14yBHC7Nkh6YkDneTauqkPSNEw2YVCvZNr_RplMvm40I/edit?tab=t.0#heading=h.68h9xnjc0rt3) envisages these as dot-separated (`message.create` etc) rather than snake cased. I don't really have a strong opinion on style, but did you decide to change to snake-cased for a particular reason?

Reply:
that's a good point -- @lmars shouldn't WithInteractionFields string action be dot-separated pr your proposal?

Reply:
As discussed in the recent standup, I will stick with SCREAMING_SNAKE, enum values are up to the SDK implemeter, but I'll go with `message.create`. As for the Spec, I will only include the enum names, not their values.